index/handlers.js

```diff
@@ -108,6 +108,8 @@ Handlers.prototype.completed = function(message) {
     return that.routeRegexp.test(route);
   }).map(function(route) {
     return that.routeRegexp.exec(route)[1];
+  }).filter(function(namespace) {
+    return helpers.isValidNamespace(namespace);
```

Comment:
So, this should provide us a list of namespaces that insertTask shouldn't choke on and return 400, but do we inform the user at all that some of their index routes might not be used while others might be?